src/lwc/geListView/geListView.js

```diff
@@ -149,15 +151,17 @@ export default class geListView extends LightningElement {
     }
 
     get isShowStandardFooter() {
-        const showFooter =
-            this.showStandardFooter === 'true' ||
+        return this.showStandardFooter === 'true' ||
             this.showStandardFooter === true ||
```

Comment:
Where does this.showStandardFooter get set? Also, why do we need to check against 3 values?

Reply:
I think we're compensating for the fact that @api values come across as Strings if they're set directly in the HTML for the containing component, but could be Booleans if they're bound to a variable.

On line 59 and 93 of geTemplates.html, I set the `show-standard-footer` attribute but don't give it a value, which [ends up defaulting it to Boolean true](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/js_props_boolean). As long as we're consistent about doing Boolean properties like this, I agree with dropping the extra comparisons here.

Reply:
Does this mean that the value will always be true?

Reply:
Yes. If we need to set it to `false`, we can omit the attribute entirely. And if we need to dynamically toggle it, we bind it to a variable and make sure that var is always a Boolean.